src/query/block/column.go

```diff
@@ -237,6 +237,8 @@ func (cb ColumnBlockBuilder) PopulateColumns(size int) {
 	for i := range cb.block.columns {
 		cb.block.columns[i] = column{Values: cols[size*i : size*(i+1)]}
 	}
+
+	cb.block.seriesMeta = make([]SeriesMeta, size)
```

Comment:
Should we add test coverage that this is always size of cols now?  Seems like was just missing before?

Reply:
Yeah, luckily the new test fails without this statement :+1: